js/scripts/zone_loaders/village_zone_loaders.js

```diff
@@ -210,7 +218,7 @@ const loadVillageMain = () => {
         ASSET_MGR.queueDownload(PapaChad.SPRITESHEET);
         ASSET_MGR.queueDownload('./sprites/mama_chad_trapped.png');
         ASSET_MGR.queueDownload(Wizard.SPRITESHEET);
-
+        ASSET_MGR.queueDownload(Slime.SPRITESHEET);
```

Comment:
Was this the issue you ran into with the slimes?

Reply:
I can't tell you what my issue was tbh, but copy pasting some other code and editing it worked.